client/src/components/Applications/AppForm/AppFormUtils.js

```diff
@@ -117,3 +128,57 @@ export const getQueueValidation = (queue, app) => {
       }
     );
 };
+
+/**
+ * Get corrected values for a new queue
+ *
+ * Check values and if any do not work with the current queue, then fix those
+ * values.
+ *
+ * @function
+ * @param {Object} values
+ * @returns {Object} updated/fixed values
+ */
+export const getFixedValuesForUpdatedQueue = (app, values) => {
```

Comment:
nitpicky, but we're not "fixing" values so much as we are "updating". What do you think?

Reply:
Sounds good 👍